apps/omg_eth/test/omg_eth/signature_test.exs

```diff
@@ -0,0 +1,105 @@
+# Copyright 2018 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Eth.SignatureTest do
+  @moduledoc """
+  Tests that elixir-generated signatures can be successfully verified by contract code.
+  NOTE: These are integration tests as they require Ethereum node running and contract deployment,
+  however it deployed contract once for all tests and then calls a function on it which should be
+  quick enough so we can afford it runs with unit tests.
+  """
+
+  alias OMG.DevCrypto
+  alias OMG.Eth
+  alias OMG.State.Transaction
+  alias OMG.TestHelper
+
+  use ExUnitFixtures
+  use ExUnit.Case, async: false
+
+  @alice TestHelper.generate_entity()
+  @bob TestHelper.generate_entity()
+  @eth OMG.Eth.RootChain.eth_pseudo_address()
+  @token TestHelper.generate_entity().addr
+
```

Comment:
this definitely requires an `:integration` and `:common` module tags. Reflecting on the note from moduledoc, unit tests should be available also on machines without `eth_node`.